### PR TITLE
PlayerHUD uses flexbox

### PR DESCRIPTION
**Pull request by Steedie**
## What
PlayerHUD now uses flexbox instead of absolute positioning:
image
image

## Resolves
- Resolves: #185

### CODE REVIEW

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 1]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
                     <div
                         style={{
-                            position: 'absolute',
-                            top: '30px',
-                            left: '50%',
-                            transform: 'translateX(-50%)',
-                            width: '50%',
-                            maxWidth: '300px',
+                            flex: 1,
+                            maxWidth: '33%',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx:131]: Without it, the chat text affects the flex spacing:
image

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 2]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
+                    ) : null}
+                </div>
+                {/* Bottom row */}
+                <div
+                    style={{
+                        display: 'flex',
+                        flexGrow: 0,
+                        flexShrink: 1,
+                        flexBasis: 'auto',
+                        alignSelf: 'auto',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx]: Removed: https://github.com/playmint/playerchain-demo/pull/208/commits/d096f65050df2433433e53d3764db8caf817dd16#diff-c0b9400cf5566b795bb5b64fce028b269632fb1bfdbf0171c045d079c3e25dbeL121-L122

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 3]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
                     </div>
+
                     <div
                         style={{
-                            position: 'absolute',
-                            bottom: '30px',
-                            right: '30px',
-                            width: 'auto',
+                            flex: 1,
+                            fontSize: '20px',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx]: Fixed: https://github.com/playmint/playerchain-demo/pull/208/commits/d096f65050df2433433e53d3764db8caf817dd16#diff-c0b9400cf5566b795bb5b64fce028b269632fb1bfdbf0171c045d079c3e25dbeL146-R141

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 4]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
+                <div
+                    style={{
+                        display: 'flex',
+                        flexGrow: 0,
+                        flexShrink: 1,
+                        flexBasis: 'auto',
+                        alignSelf: 'auto',
+                        justifyContent: 'space-between',
+                        alignItems: 'flex-end',
+                        padding: '30px',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx]: Fixed: https://github.com/playmint/playerchain-demo/pull/208/commits/d096f65050df2433433e53d3764db8caf817dd16#diff-c0b9400cf5566b795bb5b64fce028b269632fb1bfdbf0171c045d079c3e25dbeL125-R120

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 5]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
+                </div>
+                {/* Middle */}
                 <div
                     style={{
                         display: 'flex',
-                        alignContent: 'center',
                         justifyContent: 'center',
                         alignItems: 'center',
+                        height: '100%',
+                        padding: '30px',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx]: Fixed: https://github.com/playmint/playerchain-demo/pull/208/commits/d096f65050df2433433e53d3764db8caf817dd16#diff-c0b9400cf5566b795bb5b64fce028b269632fb1bfdbf0171c045d079c3e25dbeL101-R98

File: src/examples/spaceshooter/renderer/PlayerHUD.tsx [hunk 6]
```diff
@@ -63,89 +63,114 @@ export default memo(function PlayerHUD({
...
-            {remaining !== 0 && (
+                {/* Top row */}
+                <div
+                    style={{
+                        display: 'flex',
+                        flexGrow: 0,
+                        flexShrink: 1,
+                        flexBasis: 'auto',
+                        alignSelf: 'auto',
+                        padding: '30px',
```
[the remark this answers is not in the document]
Comment (Steedie) [reply src/examples/spaceshooter/renderer/PlayerHUD.tsx]: fixed: https://github.com/playmint/playerchain-demo/pull/208/commits/d096f65050df2433433e53d3764db8caf817dd16#diff-c0b9400cf5566b795bb5b64fce028b269632fb1bfdbf0171c045d079c3e25dbeL86-R86

**Review by chrisfarms — APPROVED** [review]

**Pull request merged**